apps/condo/domains/ticket/constants.js

```diff
@@ -18,6 +18,21 @@ const TICKET_STATUS_TYPES = [
     CLOSED_STATUS_TYPE,
 ]
 
+/*
```

Comment:
it's unexpected for me to see the docs here %) it's better to move it inside schema